[2025-07-29T22:17:00.334+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: table_dag.submit_tablea scheduled__2025-05-30T00:00:00+00:00 [queued]>
[2025-07-29T22:17:00.339+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: table_dag.submit_tablea scheduled__2025-05-30T00:00:00+00:00 [queued]>
[2025-07-29T22:17:00.340+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-07-29T22:17:00.350+0000] {taskinstance.py:1380} INFO - Executing <Task(PinotTableSubmitOperator): submit_tablea> on 2025-05-30 00:00:00+00:00
[2025-07-29T22:17:00.353+0000] {standard_task_runner.py:57} INFO - Started process 1483 to run task
[2025-07-29T22:17:00.355+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'table_dag', 'submit_tablea', 'scheduled__2025-05-30T00:00:00+00:00', '--job-id', '812', '--raw', '--subdir', 'DAGS_FOLDER/table_dag.py', '--cfg-path', '/tmp/tmp9gek59hr']
[2025-07-29T22:17:00.358+0000] {standard_task_runner.py:85} INFO - Job 812: Subtask submit_tablea
[2025-07-29T22:17:00.388+0000] {task_command.py:415} INFO - Running <TaskInstance: table_dag.submit_tablea scheduled__2025-05-30T00:00:00+00:00 [running]> on host 931bde20eb1e
[2025-07-29T22:17:00.427+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='GGCODE' AIRFLOW_CTX_DAG_ID='table_dag' AIRFLOW_CTX_TASK_ID='submit_tablea' AIRFLOW_CTX_EXECUTION_DATE='2025-05-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-30T00:00:00+00:00'
[2025-07-29T22:17:00.435+0000] {pinot_table_operator.py:28} ERROR - Pinot table failed to submit to Apache Pinot! 409 - {"code":409,"error":"Table config for account_dim_OFFLINE already exists. If this is unexpected, try deleting the table to remove all metadata associated with it."}
[2025-07-29T22:17:00.435+0000] {pinot_table_operator.py:32} ERROR - An error occured: Table submission failed with status code 409
[2025-07-29T22:17:00.440+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=table_dag, task_id=submit_tablea, execution_date=20250530T000000, start_date=20250729T221700, end_date=20250729T221700
[2025-07-29T22:17:00.453+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-07-29T22:17:00.463+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
